.pre-commit-config.yaml

```
@@ -37,7 +38,7 @@ repos:
   rev: 6.3.0
   hooks:
   - id: pydocstyle
-    exclude: ^tests/.*
+    exclude: (tests/|migrations/)
```

Don't drop the beginning of the line marker. Also, it's usually better to multi-line more complex regexes:
```suggestion
    exclude: >-
      (?x)
      ^
      (
      tests
      |migrations
      )/.*
      $
```
Also, do you really need to exclude `tests` here? They deserve to be linted just like other files as they aren't auto-generated.

P.S. Consider dropping a separate `pydocstyle` in favor of the `flake8-docstrings` plugin that uses it under the hood.

---

Marking as resolved.